### DISCUSSION TITLE
[Bug]: Sublabel not shown in Group view

### DISCUSSION THREAD

**Discussion by Plawasan** [blakeblackshear/frigate#13142] [Report a Bug] [answered]
### Describe the problem you are having

I've just added ALPR to my setup, I've noticed that when I'm in the "home" view with all my cameras, the label (Car) and sublabel both show up when I over over the two icons.
![image](https://github.com/user-attachments/assets/c1388cf3-f5ba-4d55-8aeb-285c1b2c3312)

In Group view however I only see one icon for this camera (license plate) and the popup is empty - highlighted in red.

![image](https://github.com/user-attachments/assets/b1d81141-9cc4-4ecb-b5ed-f10690699371)

### Steps to reproduce

Add a camera to a group, generate a sublabel, hover over the icons.

### Version

0.14.0-da913d8

### Frigate config file

```yaml
Will provide if needed for this particular issue
```

### Relevant log output

```shell
N/A
```

### Operating system

Proxmox

### Install method

Docker Compose

### Network connection

Wired

### Camera make and model

UNV

### Any other information that may be helpful

Chrome Version 127.0.6533.120 (Official Build) (64-bit) on Windows

**Comment by NickM-27** [comment]
I am not able to reproduce this, it shows the same either way
<img width="129" alt="Screenshot 2024-08-17 at 10 37 12 AM" src="https://github.com/user-attachments/assets/51558326-252a-4c46-89c0-d31e7fc7889c">

**Reply by Plawasan** [reply]
Do you see the actual label/sublabel when you hover over the icons?

**Reply by Plawasan** [reply]
Seems to me like the content is there, but since the popup opens above the icons, it's hidden below the elements above it. When I zoom into that corner of the camera image, the popup shows up below the icons and then the content is visible..

EDIT: it's only an issue when there are Alert thumbnails above the camera views, when I mark them as reviewed, the popup shows up correctly even in the group view.

**Reply by NickM-27** [reply]
I see what you mean, the screenshot didn't convey it as well in light theme.

**Reply by NickM-27** [reply, marked as answer]
this has been fixed 👍
